src/AspNet.Security.OAuth.JumpCloud/JumpCloudAuthenticationOptions.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.JumpCloud;
+
+/// <summary>
+/// Defines a set of options used by <see cref="JumpCloudAuthenticationHandler"/>.
+/// </summary>
+public class JumpCloudAuthenticationOptions : OAuthOptions
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="JumpCloudAuthenticationOptions"/> class.
+    /// </summary>
+    public JumpCloudAuthenticationOptions()
+    {
+        ClaimsIssuer = JumpCloudAuthenticationDefaults.Issuer;
+        CallbackPath = JumpCloudAuthenticationDefaults.CallbackPath;
+
+        Scope.Add("openid");
+        Scope.Add("profile");
```

Comment:
In general, we try to limit the default scopes to the absolute required ones. For a standard-compliant OIDC provider, only `openid` should be required.